src/test/resources/Regression.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Regression" parallel="methods" thread-count="2">
+    <parameter name="BaseURL" value="https://qa.koel.app/"/>
+    <test name="Example Test" preserve-order="false">
+        <classes>
+            <class name="LoginTests"/>
+            <class name="BaseTest"/>
+        </classes>
```

Comment:
Please add more class in Regression.xml and run

For example

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/2fc629c2-1a3f-4d46-8285-1e47029d0155)